code/modules/paperwork/paper_bundle.dm

```diff
@@ -140,14 +143,12 @@
 /obj/item/paper_bundle/attack_self(mob/user as mob)
 	show_content(user)
 	add_fingerprint(usr)
-	update_icon()
 	return
 
 /obj/item/paper_bundle/Topic(href, href_list)
 	if(..())
 		return
-
-	if((src in usr.contents) || (istype(src.loc, /obj/item/folder) && (src.loc in usr.contents)))
+	if(src in usr.contents)
```

Comment:
does this break turning paper stacks in folders?

Reply:
Turning papers never worked in folders or clipboards, I originally added support for it in this PR but with the set_machine changes that AA made, it won't be able to be supported anymore.